scripts/migration/migrate_personal_to_profile_websites.py

```diff
@@ -0,0 +1,121 @@
+
+import logging
+
+from modularodm import Q
+from nose.tools import *
+from tests.base import OsfTestCase
+
+from website import models
+from website.app import init_app
+from scripts import utils as scripts_utils
+from tests.factories import AuthUserFactory
+
+
+logger = logging.getLogger(__name__)
+
+
+def main():
+    init_app(routes=False)
+    dry_run = 'dry' in sys.argv
+    if not dry_run:
+        scripts_utils.add_file_logger(logger, __file__)
+    logger.info("migrating personal to profileWebsites")
+            
+    for user in get_users_with_personal_websites():
+        logger.info(repr(user))
+        logger.info(repr(user.social))
+        if not user.social.get('profileWebsites', None):
+            user.social['profileWebsites'] = []
+            if user.social.get('personal'):
+                migrate_personal_to_profile_websites(user)
+        logger.info(repr(user.social))
+        if not dry_run:
+            user.save()
+
+
+def get_users_with_personal_websites():
+    return models.User.find(
+        Q('social', 'ne', {})
+    )
+
+
+def migrate_personal_to_profile_websites(user):
+    user.social['profileWebsites'].append(user.social.get('personal'))
+
+
+class TestMigrateProfileWebsites(OsfTestCase):
+
+    def setUp(self):
+        super(TestMigrateProfileWebsites, self).setUp()
+        self.user_one = AuthUserFactory.build(
+            fullname='Martin Luther King',
+            social=dict(
+                github='userOneGithub',
+                scholar='userOneScholar',
+                personal='http://www.useronewebsite.com',
+                twitter='userOneTwitter',
+                linkedIn='userOneLinkedIn',
+                impactStory='userOneImpactStory',
+                orcid='userOneOrcid',
+                researcherId='userOneResearcherId',
+            ),
+        )
+        self.user_one.save()
+
+        self.user_two = AuthUserFactory.build(
+            fullname='el-Hajj Malik el-Shabazz',
+            social=dict(
+                github='userTwoGithub',
+                scholar='userTwoScholar',
+                profileWebsites=['http://www.usertwowebsite.com'],
+                twitter='userTwoTwitter',
+                linkedIn='userTwoLinkedIn',
+                impactStory='userTwoImpactStory',
+                orcid='userTwoOrcid',
+                researcherId='userTwoResearcherId'
+            )
+        )
+        self.user_two.save()
+        self.user_three = AuthUserFactory()
+
+    def test_get_users_with_personal_website(self):
+        users = []
+        for user in get_users_with_personal_websites():
+            users.append(user)
+
+        assert_equal(self.user_one._id, users[0]._id)
+        assert_equal(self.user_two._id, users[1]._id)
+        assert_equal(len(users), 2)
+
+    def test_migrate_profile_websites(self):
+        main()
+        self.user_one.reload()
+        assert_equal(self.user_one.fullname, 'Martin Luther King')
```

Comment:
Since this function is just testing migrating profile websites, I think you can remove the assert statements for the other social fields and the user's given, middle and family name.